packages/contracts-bedrock/contracts/test/LibClock.t.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import { Test } from "forge-std/Test.sol";
+import { LibClock } from "../dispute/lib/LibClock.sol";
+import "../libraries/DisputeTypes.sol";
+
+/**
+ * @notice Tests for `LibClock`
+ */
```

Comment:
I think test contract docstrings aren't super useful and just add LOC. I'd prefer if we removed at least this top level comment

Reply:
ye it's not a big deal either way, we can leave